Code/Source/svFSI/mat_fun_carray.h

```diff
@@ -458,6 +458,15 @@ double norm(const Vector<double>& u, const double v[N])
   return norm;
 }
 
+template <size_t N>
+double norm(const Vector<double>& u, const Vector<double>& v)
```

Comment:
There is already a `norm` function in `utils.cpp`.

Reply:
I am removing this function.